accounts/forms.py

```diff
@@ -287,23 +268,6 @@ class EmailResetForm(forms.Form):
     email = forms.EmailField(label=_("New e-mail address"), max_length=75)
     password = forms.CharField(label=_("Your password"), widget=forms.PasswordInput)
 
-    def __init__(self, *args, **kwargs):
-        # Using init function to pass user variable so later we can perform check_password in clean_password function
-        self.user = kwargs.pop('user', None)
-        super(EmailResetForm, self).__init__(*args, **kwargs)
-
-    def clean_email(self):
-        email = self.cleaned_data["email"]
-        try:
-            get_user_by_email(email)
-            raise forms.ValidationError(_("A user using that email address already exists."))
-        except User.DoesNotExist:
-            pass
-        return email
-
-    def clean_password(self):
```

Comment:
Why is this clean function also removed?

Reply:
I think this was removed by mistake, I will add it again